mono/metadata/boehm-gc.c

```diff
@@ -959,371 +959,9 @@ mono_gc_get_restart_signal (void)
 }
 
 #if defined(USE_COMPILER_TLS) && defined(__linux__) && (defined(__i386__) || defined(__x86_64__))
```

Comment:
Remove from ‘#if’ to/including ‘#else’ instead.

Reply:
Now that I have bigger screen -- this isn't terrible. Just a tiny excess remains.